styles.css

```diff
@@ -147,5 +446,154 @@ section {
     background-position: 100% 100%;
   }
 
- 
+  .container h1 {
+    font-family: 'Poppins' "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .container p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .work_content {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    height: 496px;
+  }
+
+  .work_content img {
+    float: left;
+    width: 48%;
+  }
+
+  #pic2,
+  #pic4 {
+    flex-direction: row-reverse;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] Great work :clap: but still little bit issues in your design if you check the languages section it's now shown down not the right side so kindly fix it as well. Check below :point_down: the comparison between your design and Figma design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-04-07 02-27-13](https://user-images.githubusercontent.com/45226848/230499435-3c8d2738-b7e9-4f59-9745-8f268af730a4.png)

**Figma Design**
![Figma design](https://user-images.githubusercontent.com/102699958/230458000-47622cf2-9c23-433a-b3ee-7b967202b742.png)